From a98bd86292ddd34824e0ba36edb06a6f11184cc9 Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Thu, 18 Jan 2018 13:15:57 +0900
Subject: [PATCH 2754/2848] HADOOP-15177. Update the release year to 2018.
 Contributed by Bharat Viswanadham.

(cherry picked from commit cdaf92c9f57560219b8f915a19ad8603ddf2a505)
(cherry picked from commit b0c9b89281c9264c70ae79b1e4830e0fa2411b4f)
(cherry picked from commit 17a673e740cc626f1537a8434ecce4ff7ffe3b8e)

 Conflicts:
	hadoop-common-project/hadoop-common/CHANGES.txt

Change-Id: I4465d5f4d06397e0590a5354814053b0253d054f
---
 hadoop-project/pom.xml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-project/pom.xml b/hadoop-project/pom.xml
index 409a73b..a9d827a 100644
--- a/hadoop-project/pom.xml
+++ b/hadoop-project/pom.xml
@@ -31,7 +31,7 @@
 
   <properties>
     <!-- Set the Release year during release -->
-    <release-year>2017</release-year>
+    <release-year>2018</release-year>
 
     <failIfNoTests>false</failIfNoTests>
     <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
-- 
1.7.9.5

